jbpm-audit/pom.xml

```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>org.jbpm</groupId>
     <artifactId>jbpm</artifactId>
-    <version>7.26.0-SNAPSHOT</version>
+    <version>7.27.0-SNAPSHOT</version>
```

Comment:
@ruivieira This is from commit made by Michael Biarnes, I believe you merged 2 branches. But instead I think you should have done a rebase. This would ensure that only your commits are on your PR branch. I think you need to sort this out locally and then force push to your BAPL-1123 branch.